src/Controller/HearingController.php

```diff
@@ -68,6 +69,7 @@ public function index(CaseEntity $case, HearingPostRepository $hearingPostReposi
         $form->handleRequest($request);
         if ($form->isSubmitted() && $form->isValid()) {
             $this->entityManager->flush();
+            $this->addFlash('success', new TranslatableMessage('Hearing post updated', [], 'case'));
```

Comment:
Should just say `Hearing updated`, as it is to do with the general state of a `Hearing`, not a `HearingPost`.